scalalib/src/mill/scalalib/TestRunner.scala

```diff
@@ -19,87 +19,103 @@ import scala.collection.mutable
-        args = args(3) match{ case "" => Nil case x => x.split(" ").toList }
-      )(new PrintLogger(
-        args(5) == "true",
-        if(args(5) == "true") Colors.Default
-        else Colors.BlackWhite,
-        System.out,
-        System.err,
-        System.err
-      ))
+        args = args(3) match {
+          case "" => Nil
+          case x => x.split(" ").toList
+        }
+      )(
+        new PrintLogger(
+          args(5) == "true",
+          if (args(5) == "true") Colors.Default
+          else Colors.BlackWhite,
+          System.out,
+          System.err,
+          System.err
+        ))
       val outputPath = args(4)
 
       ammonite.ops.write(Path(outputPath), upickle.default.write(result))
-    }catch{case e: Throwable => 
-      println(e)
-      e.printStackTrace()
+    } catch {
+      case e: Throwable =>
+        println(e)
+        e.printStackTrace()
     }
     // Tests are over, kill the JVM whether or not anyone's threads are still running
     // Always return 0, even if tests fail. The caller can pick up the detailed test
     // results from the outputPath
     System.exit(0)
   }
-  def apply(frameworkName: String,
-            entireClasspath: Agg[Path],
-            testClassfilePath: Agg[Path],
-            args: Seq[String])
-           (implicit ctx: LogCtx): (String, Seq[Result]) = {
-    Jvm.inprocess(entireClasspath, classLoaderOverrideSbtTesting = true, cl => {
-      val framework = cl.loadClass(frameworkName)
-        .newInstance()
-        .asInstanceOf[sbt.testing.Framework]
-
-      val testClasses = runTests(cl, framework, testClassfilePath)
-
-      val runner = framework.runner(args.toArray, args.toArray, cl)
-
-      val tasks = runner.tasks(
-        for ((cls, fingerprint) <- testClasses.toArray)
-          yield new TaskDef(cls.getName.stripSuffix("$"), fingerprint, true, Array(new SuiteSelector))
-      )
-      val events = mutable.Buffer.empty[Event]
-      for (t <- tasks) {
-        t.execute(
-          new EventHandler {
-            def handle(event: Event) = events.append(event)
-          },
-          Array(
-            new Logger {
+  def apply(
+      frameworkName: String,
+      entireClasspath: Agg[Path],
+      testClassfilePath: Agg[Path],
+      args: Seq[String])(implicit ctx: LogCtx): (String, Seq[Result]) = {
+    Jvm.inprocess(
+      entireClasspath,
+      classLoaderOverrideSbtTesting = true,
+      cl => {
+        val framework = cl
+          .loadClass(frameworkName)
+          .newInstance()
+          .asInstanceOf[sbt.testing.Framework]
+
+        val testClasses = runTests(cl, framework, testClassfilePath)
+
+        val runner = framework.runner(args.toArray, args.toArray, cl)
+
+        val tasks = runner.tasks(
+          for ((cls, fingerprint) <- testClasses.toArray)
+            yield
+              new TaskDef(cls.getName.stripSuffix("$"), fingerprint, true, Array(new SuiteSelector))
```

Comment:
This part looks pretty gross; is there some better way scalafmt can handle this? ideally I'd want `for` and `yield` to be vertical-aligned, and `new TaskDef` to be on the same line as `yield` (the arguments can be on the same line or following-lines)